### PR TITLE
Fix default network and DNS for compute nodes on Warewulf 4

### PR DESCRIPTION
**Pull request by middelkoopt**
Add default networking to node profiles   
    * Add netmask, gateway, and dns at the node profile level
    * Add internal_network, ipv4_gateway, dns_servers across
      input.local.template and inputs.tex

### CODE REVIEW

**Comment by middelkoopt** [comment]
Hold on, something's not right with this PR/commit.

**Comment by middelkoopt** [comment]
PR Fixed, something went wonky while rebasing against the latest 3.x.

File: docs/recipes/install/rocky9/input.local.template [hunk 1]
```diff
@@ -21,6 +21,9 @@ sms_eth_internal="${sms_eth_internal:-eth1}"
 # Subnet netmask for internal cluster network
 internal_netmask="${internal_netmask:-255.255.0.0}"
 
+# Subnet network for internal cluster network
+internal_netmask="${internal_network:-172.16.0.0}"
```
Comment (adrianreber) [inline docs/recipes/install/rocky9/input.local.template]: Copy paste typo here.

File: docs/recipes/install/common/warewulf4_setup_centos.tex [hunk 2]
```diff
@@ -5,10 +5,6 @@
 [sms](*\#*) ip link set dev ${sms_eth_internal} up
 [sms](*\#*) ip address add ${sms_ip}/${internal_netmask} broadcast + dev ${sms_eth_internal}
 
-# Compute the network address for the internal network
-[sms](*\#*) internal_cidr=$(netmask ${sms_ip}/${internal_netmask})
```
Comment (adrianreber) [inline docs/recipes/install/common/warewulf4_setup_centos.tex:9]: This means you can also remove the netmask package installation in the recipe.

**Comment by middelkoopt** [comment]
Good eye. Fixes applied.  I forced pushed this time since there were small changes, let me know if I should stick to delta's and squash a t the end.

**Comment by adrianreber** [comment]
Force pushing is almost always the right solution from my point of view. Except you want to split commits for some reason. I would not push fixup commits to a PR. But that is just me.

**Pull request merged**